client/my-sites/themes/themes-selection.jsx

```diff
@@ -159,9 +161,13 @@ class ThemesSelection extends Component {
 		return (
 			<div className="themes__selection">
 				<QueryThemes query={ query } siteId={ source } />
+				{ this.props.forceWpOrgSearch && source !== 'wporg' && (
```

Comment:
Non-blocking question why is this `forceWpOrgSearch` boolean required? I have checked the occurrences of `themes-selection` but I don't find any that use the default so in my findings this is always `true`

Reply:
There are quite a few `ConnectedThemesSelection` occurrences, and not all of them should query for WP.org themes, that's why I added this prop.